src/multimanifestmodule.ts

```diff
@@ -8,7 +8,8 @@ import { stackanalysismodule } from './stackanalysismodule';
 import { authextension } from './authextension';
 import { StatusMessages } from './statusMessages';
 import { DependencyReportPanel } from './dependencyReportPanel';
-import { Apiendpoint } from './apiendpoint';
+import { Config } from './Config';
+import { config } from 'process';
```

Comment:
@dvandra can you take a look, is this import required?

Reply:
No, I missed this to remove.